scipy/signal/lti_conversion.py

```diff
@@ -384,6 +384,36 @@ def cont2discrete(system, dt, method="zoh", alpha=None):
     approximation is based on [2]_ and [3]_, the First-Order Hold (foh) method
     is based on [4]_.
 
+    Examples
+    --------
+    We can transform a continuous state-space system to a discrete one:
+
+    >>> import matplotlib.pyplot as plt
+    >>> from scipy.signal import cont2discrete, lti, dlti, dstep
+
+    Define a continious state-space system.
+
+    >>> A = np.array([[0, 1],[-10, -3]])
```

Comment:
The plant model needs float numbers otherwise it doesn't work for some reason I can't find right now. There must be an issue somewhere hence we need to introduce some dots in the data 

```
A = np.array([[0, 1],[-10., -3]])
B = np.array([[0],[10.]])
C = np.array([[1., 0]])
D = np.array([[0.]])
l_system = lti(A, B, C, D)
t, x = l_system.step(T=np.linspace(0,5,100))
```

should do it

Reply:
Thanks. I fixed it.